docs/madr/decisions/080-inbound-policies-matches.md

```diff
@@ -117,6 +97,50 @@ spec:
             value: "spiffe://trust-domain.mesh/"
 ```
 
+##### Merging
+
+Normally in Kuma policies mergeable arrays override each other, for example:
+
+```
+name: policy-1
+spec:
+  default:
+    myArray: [1,2,3]
+---
+name: policy-1
+spec:
+  default:
+    myArray: [4]
+```
+
+results in `spec.default.myArray: [4]`.
+
+Switching array merging behaviour from overriding to concatenation normally requires prefixing fields with `append`:
+
+```
+name: policy-1
+spec:
+  default:
+    appendArray: [1,2,3]
+---
+name: policy-1
+spec:
+  default:
+    appendArray: [4]
+```
+
+results in `spec.default.appendArray: [1,2,3,4]`.
+
+Instead of naming fields `appendDeny` and `appendAllow` we need to add a new value `concat` for the existing `policyMerge` struct tag:
```

Comment:
Is aesthetics the only reason for this? Honestly I don't mind it being called `append*` - that way the api is consistent (no need to explain to the user, there are some fields that append that have `append` prefix and some don't)

Reply:
Discussed offline to go with struct tag instead of `append`